packages/core/src/styles/components/_threshold.scss

```diff
@@ -0,0 +1,49 @@
+.#{$prefix}--#{$charts-prefix}--threshold {
+  line {
+    stroke: $red-50;
+    stroke-width: 1;
+    stroke-dasharray: 4;
+    cursor: pointer;
+  }
```

Comment:
```suggestion
    pointer-events: none;
  }
```

when the cursor is directly over the `<line>` you get a weird flicker because its picking that up instead of the `threshold-hoverable-area`

Reply:
Thanks, good catch 🙂